src/platform/common/featureManager.ts

```diff
@@ -84,10 +84,7 @@ export class FeatureManager implements IFeaturesManager {
 
     private _updateFeatures() {
         const kernelPickerType =
-            this.workspace.getConfiguration('notebook.kernelPicker').get('type') === 'mru' ||
-            this.configService.getSettings().kernelPickerType === 'Insiders'
```

Comment:
@rebornix didn't you want to use a single setting for this?
I don't see why we need both VS Code and Jupyter setting, if we want to use just one setting

Reply:
@DonJayamanne yes, we should only use one from VS Code.